docs/sphinx/reference-scenes.rst

```diff
@@ -298,9 +298,28 @@ Scene Item Functions
 
 ---------------------
 
+.. function:: obs_sceneitem_t *obs_scene_sceneitem_from_source(obs_scene_t *scene, obs_source_t *source)
+
+   :return: The sceneitem associated with a source in a scene
+
+---------------------
+
+.. function:: void obs_sceneitem_set_id(obs_sceneitem_t *item);
 .. function:: int64_t obs_sceneitem_get_id(const obs_sceneitem_t *item)
 
-   :return: The unique numeric identifier of the scene item.
+   This shouldn't really be used.
```

Comment:
Sort of an odd comment.  Should something else be used?  Should this be deprecated?

Reply:
Maybe this should just be removed from the documentation. This is a method that is used internally for undo/redo, but it is dangerous and can very easily break things if not used correctly. I was thinking that it should be in the docs, but with a warning that it should not be used.

Reply:
Hmm.  There doesn't appear to be any C specific syntax for marking a private API if a function itself.  There's `:meta private:` for Python, but I don't know if it would make sense to (ab)use that here.  If they are actually exposed and you want to document them, yeah, I'd make it clear they are meant to be _internal_ functions.